.github/workflows/build-test.yml

```diff
@@ -451,7 +451,7 @@ jobs:
           python ./pythonFiles/install_debugpy.py
           conda env update --file ./build/conda-functional-requirements.yml
           python -m pip install --upgrade -r ./build/conda-functional-requirements.txt
-          conda install pytorch torchvision torchaudio cpuonly -c pytorch
+          conda install pytorch==1.8.0 torchvision==0.9.0 torchaudio==0.8.0 cpuonly -c pytorch
```

Comment:
Functional tests are failing because we can't install pytorch.
This didn't really fix it, but I think its safer to stay in a constant stable version and update it when necessary, instead of risking pytorch coming out with a broken release.